src/coreclr/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -758,6 +758,9 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
         case InstructionSet_AVX512BW_VL:
         case InstructionSet_AVX512VBMI:
         case InstructionSet_AVX512VBMI_VL:
+        case InstructionSet_AVX10v1:
+        case InstructionSet_AVX10v1_V256:
+        case InstructionSet_AVX10v1_V512:
```

Comment:
Looks like further down in `genHWIntrinsic_R_RM` there's some handling around `if (compiler->compOpportunisticallyDependsOn(InstructionSet_AVX512BW_VL))` and `else if (compiler->compOpportunisticallyDependsOn(InstructionSet_AVX512F_VL))` that needs to account for Avx10v1

Reply:
I have made the suggested change.

Reply:
I have made the changes where all the ```VL``` instruction set checks needs to replaced. This change is included in my latest commit.